rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -51,6 +51,12 @@ paths:
                   format: binary
                   minLength: 1
                   maxLength: 14316557655
+                correlationId:
```

Comment:
I think we should make this required so that we don't need to implement both endpoints. We don't have a need right now to ingest without a correlation id.

Reply:
@jhunzik Reminder to make sure that the minor version is appropriately bumped for this new required param.

Reply:
@paouelle https://github.com/connexta/ion-ingest-api/pull/52#discussion_r313631480 yes